src/Layout/redist/targets/Version.targets

```diff
@@ -1,22 +1,4 @@
 <Project>
-  <Target Name="GetCoreSdkGitCommitInfo">
-    <Exec Command="git rev-list --count HEAD"
-          ConsoleToMSBuild="true"
-          Condition=" '$(GitCommitCount)' == '' ">
-      <Output TaskParameter="ConsoleOutput" PropertyName="GitCommitCount" />
-    </Exec>
-
-    <Exec Command="git rev-parse HEAD"
-          ConsoleToMSBuild="true"
-          Condition=" '$(GitCommitHash)' == '' ">
-      <Output TaskParameter="ConsoleOutput" PropertyName="GitCommitHash" />
```

Comment:
GitCommitHash is still used in `src\Layout\redist\targets\GenerateLayout.targets`

Reply:
Ahh right good catch.